web/api/v1/api.go

```diff
@@ -175,6 +176,7 @@ type API struct {
 	config                func() config.Config
 	flagsMap              map[string]string
 	ready                 func(http.HandlerFunc) http.HandlerFunc
+	option                Options
```

Comment:
`option` -> `options`, but see my other comment about naming it generally more specifically.

Reply:
we already have a func `options` that implements an interface. Hence, it would be conflicting. how about `opts`?

Reply:
Yeah, but `options` was too generic anyway, so now it'd be `globalURLOptions`?

Reply:
sure.